### ISSUE TITLE
AttributeError: '_EarlyLogger' object has no attribute 'warn'

### ISSUE THREAD

**Issue by nivha** [wandb/client#1532]
Reproducible in my notebook:

```
import os
import wandb
os.environ['WANDB_SILENT'] = 'true'
wandb.init(project='metrics')
for epoch in range(2):
    for batch in range(2):
        wandb.log({"loss": 1.0}, step=batch)
```

Results in the error in the title

**Comment by ariG23498** [comment]
Hey @nivha 
It would be great if you posted the entire stack trace for people to take a look at.

**Comment by raubitsj** [comment]
This should have been fixed with 0.10.11?    Can you verify it is still a problem with the latest release.

**Comment by miraodasilva** [comment]
I had this exact issue on 0.10.8. I can confirm that, for me, upgrading to 0.10.11 solved it.

**Comment by nivha** [comment]
Hey @raubitsj indeed fixed in 0.10.11 as @miraodasilva said.
@ariG23498 since it's already fixed I think we can just forget about it (?)
